schwifty/iban.py

```diff
@@ -196,19 +209,19 @@ def validate(self):
 
     def _validate_characters(self):
         if not re.match(r"[A-Z]{2}\d{2}[A-Z]*", self.compact):
-            raise ValueError("Invalid characters in IBAN {}".format(self.compact))
+            raise InvalidIBANCharacters("Invalid characters in IBAN {}".format(self.compact))
```

Comment:
I think this could be the same as `InvalidStructure`, since this validation step basically checks if the IBAN is generally formatted correctly before checking the country specific format in `_validate_format`